build/publish/AzureInfo.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+   
+  <Target Name="SetAzureProps"
+          Condition=" '$(IsOrchestratedPublish)' == 'True' And '$(PB_PublishBlobFeedUrl)' != '' ">
+
+    <ParseBlobUrl BlobUrl="$(PB_PublishBlobFeedUrl)">
+      <Output TaskParameter="BlobElements" ItemName="BlobElements" />
+    </ParseBlobUrl>
+
+    <PropertyGroup>
+      <ArtifactContainerName>%(BlobElements.ContainerName)</ArtifactContainerName>
+      <ChecksumContainerName>%(BlobElements.ContainerName)</ChecksumContainerName>
+      <ArtifactCloudDropAccountName>%(BlobElements.AccountName)</ArtifactCloudDropAccountName>
+      <ChecksumCloudDropAccountName>%(BlobElements.AccountName)</ChecksumCloudDropAccountName>
+      <PublishBlobFeedUrlBlobPath>%(BlobElements.BlobPath)</PublishBlobFeedUrlBlobPath>
+      <BlobStoragePartialRelativePath>$(PublishBlobFeedUrlBlobPath.Replace('/index.json', ''))/assets</BlobStoragePartialRelativePath>
```

Comment:
Unless I'm missing something, this should include `/Sdk` (`/$(BlobStoragePartialRelativePath)`) at the end. Final publish puts everything below `assets` directly in the container in `dotnetcli` so this will give you the `Sdk` virtual dir.

Reply:
An "SDK" token does not seem consistent given that the 'Publish' URL is: 'https://dotnetfeed.blob.core.windows.net/orchestrated/20180110-13/cli/index.json' It surfactantly appears that orchestrated build is using "CLI" to detonate this 'leg' of production. I remain open to discussion.

Reply:
Final publish will take everything in
`https://dotnetfeed.blob.core.windows.net/orchestrated/20180110-13/cli/assets/` and put it at 
`https://dotnetcli.blob.core.windows.net/dotnet/`. That's why I'm saying you need the Sdk virtual dir here. Whether or not the plan is to use `PushToBlobFeed`, the artifacts still have to be in the right location per https://github.com/dotnet/core-eng/blob/master/Documentation/Orchestrated-Build/Api/publish.md#publish-runtime-and-sdk-installers-to-known-endpoints.

We can make the final publish step support additional metadata to change the target location, but it will take time and there doesn't seem to be a reason to.